src/quota_aggregator_impl.cc

```diff
@@ -137,6 +142,18 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::Quota(
   // Aggregate tokens if the cached response is positive
   if (lookup.value()->is_positive_response()) {
```

Comment:
When the cached response is negative, exceeded timeout and quota was refilled; the client request will be rejected until the response arrives. Would this be ok?